tests/test_0_API.py

```diff
@@ -19,7 +19,7 @@ def test_doc_exists():
         morphio.mut.Soma,
     ]
     for cls in classes:
-        public_methods = (method for method in dir(cls) if not method[:2] == '__')
+        public_methods = (method for method in dir(cls) if not method.startswith("_"))
```

Comment:
?

Reply:
There seems to be a new protected method that slips through that filter otherwise:
```python

    def test_doc_exists():
        classes = [
            morphio.EndoplasmicReticulum,
            morphio.MitoSection,
            morphio.Mitochondria,
            morphio.Morphology,
            morphio.Section,
            morphio.Soma,
            morphio.mut.EndoplasmicReticulum,
            morphio.mut.MitoSection,
            morphio.mut.Mitochondria,
            morphio.mut.Morphology,
            morphio.mut.Section,
            morphio.mut.Soma,
        ]
        for cls in classes:
            public_methods = (method for method in dir(cls) if not method[:2] == '__')
            for method in public_methods:
>               assert getattr(cls, method).__doc__, \
                    'Public method {} of class {} is not documented !'.format(method, cls)
E               AssertionError: Public method _pybind11_conduit_v1_ of class <class 'morphio._morphio.EndoplasmicReticulum'> is not documented !
E               assert None
E                +  where None = <instancemethod _pybind11_conduit_v1_ at 0x1062feaa0>.__doc__
E                +    where <instancemethod _pybind11_conduit_v1_ at 0x1062feaa0> = getattr(<class 'morphio._morphio.EndoplasmicReticulum'>, '_pybind11_conduit_v1_')

test_0_API.py:24: AssertionError
```

Reply:
what about exempting that exact one, just in case. I'd prefer the safety of checking for everything.

Reply:
Done.